sdk/textanalytics/ai-text-analytics/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Release History
 
-## 5.0.1 (Unreleased)
+## 5.1.0-preview.1 (Unreleased)
 
+- This release adds support for the opinion mining feature of Sentiment Analysis.
```

Comment:
we should add that this feature is only available for service version 3.1-preview.1

Reply:
done.